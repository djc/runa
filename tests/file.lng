def main(name: str, args: list[str]) -> int:
	f = open('tests/file.lng')
	print(f.read(36))
	f.close()
