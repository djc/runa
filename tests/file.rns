def main(name: str, args: array[str]):
	f = open('tests/file.lng')
	print(f.read(37))
	f.close()
