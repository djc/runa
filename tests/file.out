def main(name: str, args: list[str])
